src/main/java/pl/zaprogramujzycie/mma/utils/swagger/OpenApiConfig.java

```diff
@@ -0,0 +1,24 @@
+package pl.zaprogramujzycie.mma.utils.swagger;
+
+import io.swagger.v3.oas.models.ExternalDocumentation;
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.info.Info;
+import io.swagger.v3.oas.models.info.License;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenApiConfig {
```

Comment:
To ktory cofnig w koncu jest poprawny?

Reply:
Jedno to interface a drugie to Bean. Oba te pliki są opracowane na podstawie tutoriala jak przygotować konfigurację dla swaggera, żeby wygenerował kontrakt jak na stronie korzystając z adnotacji w kontrolerze. Na Swaggerze się nie znam, robię co każą.

Reply:
Przetestuj czy obie opcje sa potrzebne, bo zawieraja te sam zestawy danych